proposal/oscar97123/proposal.rst

```diff
@@ -0,0 +1,111 @@
+===========================
+Proposal for Bitcoin trading bot
+===========================
+
+Bitcoin continues making headlines by breaking the currency rate to USD. The current price of 
+a single bitcoin dollar exceeds $50,000 US dollars continuously for the past month. Bitcoin is 
+a great hedging instrument like gold as it has limited supply and have pros like difficult 
+to manipulate as it is decentralized.
+
+The cash/currency that we used every day are all issued by the government, it might not seems 
+to be a big problem at a first glance. However, government-based currency can have 
+quantitative easing(QE) when they think it's needed. The problem is the newly generated money ("new money") 
+didn't go into the ordinary citizen's hand, instead, it goes to the enterprise.
+
+Even though the amount of money increased after QE, but, ordinary citizens saving remains the same.
+In other words, every time when QE being introduced, we all lost money. That's why it's needed to have 
+hedging strategery no matter it's bitcoin or not.
+
+Therefore, the reason that I choose this topic is because of hedging instrument, but also for speculation.
+Because the currency fluctuation is so high that a buy and a sell can generate gains, so it is worth to
+develop a trading bot specify for it.
+
+Basic information
+=================
+
+The GitHub repository (URL) hosting your term project. <TO-BE CREATED>
+
+The project aims to develop an auto trading bot specifily for bitcoin trading in order to gain profit 
+from buying the coin at low price but selling high. 
+
+
+Problem to solve
+================
+
+The idea of a trading bot is to automatically complete the buy and sell procedure based on the instructions given.
```

Comment:
Does the price data need to be acquired real-time from the market?

If yes, how do you plan to handle the I/O?

Reply:
Yes, the price will acquired real-time from the market.
I am planning to use Binance's "REST API" as the tool of data collection.

Reply:
At the first glance, I/O will be a critical factor to your system response.  It might unnecessarily complicate the part of the work that I can grade.  You should be careful about it.